index.js

```diff
@@ -71,20 +74,27 @@ function Log(options) {
     throw new Error('No transports found');
   }
 
-  logTransports = options.transports.map(function(t) {
-    var cls = Object.keys(t)[0];
-    var opts = t[cls];
-    var Transport = winston.transports[cls];
-    return new Transport(opts);
-  });
+  if (options.rawTransports){
+    logTransports = options.transports
+  } else {
+    logTransports = options.transports.map(function(t) {
+      var cls = Object.keys(t)[0];
+      var opts = t[cls];
+      var Transport = winston.transports[cls];
+      return new Transport(opts);
+    });
+  }
 
   winstonOpts = {
     levels: sysLogLevels.levels,
     transports: logTransports,
   };
 
+  if (options.meta)
+    this.defaultMeta = options.meta
+
   winstonLog = new winston.Logger(winstonOpts);
-  this._winexConstructor = winex.factory(winstonLog, {});
+  this._winexConstructor = winex.factory(winstonLog, options.meta);
```

Comment:
The way this is implemented in `baseMeta` means that `meta.type` here will not have an effect, so I repeat specifying `meta` below in the call to winex constructor (where it does have an effect).